File: BMI.js
function Calculate_BMI(weight,height){
    BMI= (weight*10) / (height * height * .001)
    return BMI
}

function Interpret_BMI(BMI)
    {
    if (BMI < 18.5)
       alert("Underweight")
    else if (BMI >= 18.5 &&  BMI < 25)
        alert("Normal weight")
    else if (BMI >= 25 &&  BMI < 30)
        alert("Overweight ")
    else
        alert("Obese")
}

var weight=prompt("Enter your weight")
var height=prompt("Enter your height")
var BMI=Calculate_BMI(weight,height)
Interpret_BMI(BMI)

File: BMI.py
def Calculate_BMI(weight,height):
    BMI= (weight*10) / (height * height * .001)
    return BMI

def Interpret_BMI(BMI):
    if BMI < 18.5:
        print ("Underweight")
    elif BMI >= 18.5 and  BMI < 25:
        print ("Normal weight")
    elif BMI >= 25 and  BMI < 30:    
        print ("Overweight ")
    else:
        print ("Obese")

weight=int(input("Enter your weight"))
height=int(input("Enter your height"))
BMI=Calculate_BMI(weight,height)
print(BMI)
Interpret_BMI(BMI)


